assets/stylesheets/components/buttons/_buttons.scss

```diff
@@ -1,5 +1,5 @@
 .btn {
-  border-radius: $border-radius-base;
+  border-radius: 0;
```

Comment:
The value should not change here. You need to update the variable of `$border-radius-base`